File: Backend/JAVA/application/src/main/java/com/superhero/domain/model/Work.java
package com.superhero.domain.model;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;

public final class Work implements Serializable{
    @JsonProperty("occupation")
    public String occupation;

    @JsonProperty("base")
    public String baseOfOperation;
    
    public Work() {
    }

    public String getOccupation() {
        return occupation;
    }

    public void setOccupation(String occupation) {
        this.occupation = occupation;
    }

    public String getBaseOfOperation() {
        return baseOfOperation;
    }

    public void setBaseOfOperation(String baseOfOperation) {
        this.baseOfOperation = baseOfOperation;
    }
}

File: Backend/C#/SuperHero.Domain/Model/Work.cs
using SuperHero.Domain.Model.BaseModel;
using System.Text.Json.Serialization;

namespace SuperHero.Domain.Model
{
    public class Work 
    {
        [JsonPropertyName("occupation")]
        public string Occupation { get; set; }

        [JsonPropertyName("base")]
        public string BaseOfOperation { get; set; }
    }
}
